var/ramble/repos/builtin/applications/intel-mpi-benchmarks/application.py

```diff
@@ -38,14 +38,16 @@ class IntelMpiBenchmarks(SpackApplication):
     required_package('intel-mpi-benchmarks')
 
     executable('pingpong',
-               '-genv I_MPI_FABRICS=shm:tcp '
+               '-genv FI_PROVIDER=tcp '
+               '-genv I_MPI_FABRICS=shm:ofi '
```

Comment:
Maybe the `shm:ofi` portion should be a variable instead of hard-coded here. So we can parametrize it more easily. This would also let us swap out MPI versions more easily.

Reply:
I've removed all `-genv` arguments as @rfbgo suggested - users will need to set the environment variables themselves.  Please let me know if you prefer a different solution!